cves/kernel/CVE-2014-1690.yml

```diff
@@ -55,7 +55,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In attempting to account for communications using IPv6, extra lines of code
+  were deleted that contained important buffer allocation.  Implemented 
+  NetFilter Connection Tracking (NF_CONNTRACK) support for IRC protocol 
```

Comment:
Project specific stuff should be removed. People with only some programming experience may find this harder to understand.